pkg/proxy/proxy.go

```diff
@@ -85,25 +78,16 @@ type AppOverview struct {
 	Status      string `json:"status"`
 }
 
+// TODO: Rename get for getRelease
+// TODO: Remove namespace since release name is unique
 func (p *Proxy) get(name, namespace string) (*release.Release, error) {
-	list, err := p.helmClient.ListReleases(
-		helm.ReleaseListFilter(name),
-		helm.ReleaseListNamespace(namespace),
-		helm.ReleaseListStatuses(allReleaseStatuses),
-		// Get just the latest release
-		helm.ReleaseListLimit(1),
-		helm.ReleaseListSort(int32(services.ListSort_LAST_RELEASED)),
-		helm.ReleaseListOrder(int32(services.ListSort_DESC)),
-	)
+	release, err := p.helmClient.ReleaseContent(name)
```

Comment:
Good find. I am checking the current `helm` code and it seems that the signature of the function is different:

https://github.com/helm/helm/blob/master/cmd/helm/get.go#L90

Is that because a newer version?

Reply:
Is it? I believe that it is the same. It seems to be a variadic function that accepts zero or more options

```
	ReleaseContent(rlsName string, opts ...ContentOption) (*rls.GetReleaseContentResponse, error)
```

In Helm's `get` CMD it passes the version but looking at the code I noticed that if you do not pass the version, the code returns the last one which is what we want https://github.com/helm/helm/blob/7cad59091a9451b2aa4f95aa882ea27e6b195f98/pkg/tiller/release_content.go#L32

Reply:
👍 I didn't check the function definition